runtime/src/bank_forks.rs

```diff
@@ -241,22 +248,33 @@ impl BankForks {
                         );
                     }
                 }
+                snapshot_time.stop();
+                total_snapshot_ms += snapshot_time.as_ms();
                 break;
             }
         }
         if !is_root_bank_squashed {
             root_bank.squash();
```

Comment:
We should accumulating the squash timing metrics from this as well

Reply:
Yep. added